osstats_linux.go

```diff
@@ -29,6 +31,11 @@ type OSData struct {
 }
 
 func (s *Server) InitOSData() {
+	if netDevices, err := findNetDevices(); err == nil {
+		s.osData.netDevices = netDevices
+	} else {
+		log.Println("Error discovering network devices:", err)
+	}
```

Comment:
This reads slightly cleaner to me:

```go
var err error
s.osData.netDevices, err = findNetDevices()
if err != nil {
        log.Println("Error discovering network devices:", err)
}
```

Reply:
IMO it's best to avoid writing code that depends on what a function returns when `err != nil`.

Reply:
I think it's fine if the contract is clear (it could be made to be, in this case). Not a big deal, though.